_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -29,8 +29,7 @@ Before you get started, it's a good idea to discuss your new component with the
 
 To follow this tutorial ensure the following software is install on your machine:
 - [NodeJS](https://nodejs.org/en/) (Version 10 or higher)
-- [NPM](https://www.npmjs.com/get-npm) (Version 5 or higher)
-- [Bower](https://bower.io/)
+- [NPM](https://www.npmjs.com/get-npm) (Version 7 or higher)
```

Comment:
We should add v7 or higher as part of the formal spec?

Reply:
Which part of the spec do you think it should live within?

Reply:
That is a good question. It doesn't apply to services etc. so I suppose the component spec, under package management?

Reply:
Hmm, how about we spec that the `package.json` has an `engines` object with an `npm` field set to `>=7` or `'^7`?

```json
{
	"engines": {
		"npm": ">= 7"
	}
}
```

documentation for engines field --> https://docs.npmjs.com/files/package.json#engines

Reply:
I've added a **must** for the engines field in package.json.

https://github.com/Financial-Times/origami-website/pull/273/commits/1e79f1c18ed731363ff4b785bbbc3229f3f856df